scipy/stats/_continuous_distns.py

```diff
@@ -5195,6 +5195,130 @@ def _entropy(self, beta):
+   
+    %(before_notes)s
+        
+    Notes
+    -----
+    We require mass > 0 and width > 0.
+
+    The width -> 0 limit is well-defined mathematically - it's proportional
+    to a Dirac function. In the m -> 0 limit, the pdf vanishes.
+    
+    The CDF was found by Mathematica:
+
+    pdf = k/((m^2 - mass^2)^2 + mass^4*alpha^2)
+    cdf = Integrate[pdf, m]
+    
+    %(after_notes)s
+ 
+    See Also
+    --------
+    cauchy : Closely related distribution, also known as the non-relativistic Breit-Wigner 
+    halfcauchy : Closely related distribution
+        
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Relativistic_Breit-Wigner_distribution
+    
+    %(example)s
+    
+    Examples
+    --------
+    
+    Plot a Breit-Wigner distribution and random samples
+
+    >>> import matplotlib.pyplot as plt
+    >>> BW = breit_wigner(mass=125., width=10.)
+    
+    >>> mass = np.linspace(0., 200., 500)
+    >>> pdf = [BW.pdf(m) for m in mass]
+    >>> pdf_plot = plt.plot(mass, pdf)
+    
+    >>> samples = BW.rvs(1000)
+    >>> hist_plot = plt.hist(samples, normed=True, bins=100, range=[0, 200])
+    
+    >>> plt.show()
+    
+    Statistics of a Breit-Wigner distribution
+    
+    Mean and variance:
+    
+    >>> BW.stats()
+    (array(122.11538219739913), array(762.7536855273247))
+    
+    Median:
+    
+    >>> BW.median()
+    124.41711775583536
+    
+    Variance and standard deviation:
+    
+    >>> BW.var(), BW.std()
+    (762.75368552732471, 27.617995682658158)
+    
+    95 per cent interval:
+    
+    >>> BW.interval(0.95)
+    (53.719690871945822, 159.27077425799101)
+    
+    Moments:
+    
+    >>> [BW.moment(n) for n in range(1, 6)]
+    [122.11538219739913, 15674.920254744189, inf, 244138132.80994478, 80583978992.641495]
+    """
+    def _argcheck(self, mass, width):
+        return (mass > 0) & (width > 0)
+
+    def _pdf(self, m, mass, width):
+        """
+        >>> breit_wigner(mass=125., width=0.05).pdf(125.)
+        12.732396211295313
+        """
+        alpha_ = width / mass
+        gamma_ = mass**2 * (1. + alpha_**2)**0.5
+        k = 2.**(3. / 2.) * mass**2 * alpha_ * gamma_ / (pi * (mass**2 + gamma_)**0.5)
+
+        return k / ((m**2 - mass**2)**2 + mass**4 * alpha_**2)
+
+    def _cdf(self, m, mass, width):
+        """
+        >>> BW = breit_wigner(mass=125., width=0.05)
+        >>> BW.cdf(125.)
+        0.50052268648248666
+        >>> BW.cdf(1E10)
+        1.0000000000000004
+        >>> BW.cdf(0.)
+        0.0
+        """
+        alpha_ = width / mass
+        gamma_ = mass**2 * (1. + alpha_**2)**0.5
+        k = 2.**(3. / 2.) * mass**2 * alpha_ * gamma_ / (pi * (mass**2 + gamma_)**0.5)
+        
+        arg_1 = complex(-1)**(1. / 4.) / (-1j + alpha_)**0.5 * m / mass
+        arg_2 = complex(-1)**(3. / 4.) / (1j + alpha_)**0.5 * m / mass
+
+        shape_ = -1j * np.arctan(arg_1) / (-1j + alpha_)**0.5 - np.arctan(arg_2) / (1j + alpha_)**0.5
```

Comment:
I'm pretty sure this formula could be simplified further to avoid complex arithmetics.